templates/my.cnf.j2

```diff
@@ -2,10 +2,18 @@
 #password = your_password
 port = {{ mysql_port }}
 socket = {{ mysql_socket }}
+{% if ansible_os_family == 'Debian' %}
```

Comment:
Sure. Will investigate backwards-compatibility. (I should have done that anyway :-()